Homework 28.09.21/PhoneBook/ActionsOnPhoneBook.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+//struct for phone book, includes name and phone number
```

Comment:
```suggestion
// Struct for phone book, includes name and phone number
```
После `//` пробел всегда ставится, ну и все остальные комментарии у Вас с заглавной.